### DISCUSSION TITLE
Add "handle" and "grip" options in themerc

### DISCUSSION THREAD

**Discussion by RiccardoPP** [labwc/labwc#1043] [Ideas]
Hello and thanks for labwc.

I'd like to use the **window.handle** and **windows.*.grip** options with the openbox-3 themes.

> The window handle is the piece of decorations on the bottom of windows. A value of 0 means that no handle is shown. 
http://openbox.org/wiki/Help:Themes#window.handle.width

> The grips are located at the left and right sides of the window's handle.
http://openbox.org/wiki/Help:Themes#window.active.grip.bg

I think this could be a good add-on for theming and ricing labwc because it's the only Wayland compositor (apart from the full DE one) with a decent windows theming configuration (hyprland completely ignoring this, sway is more basic).

Thanks.

**Comment by johanmalm** [comment]
Happy to support.

Just to confirm - are grip and handle the bits in yellow and red respectively. 

![2023-09-03-205055_1366x768_scrot](https://github.com/labwc/labwc/assets/6955353/f0a41265-80f8-4d62-a4c7-33be50b03f2f)

**Reply by RiccardoPP** [reply]
Yes, thanks.

**Comment by AnirvanB** [comment]
Is this planned for 0.9.0?

**Reply by johanmalm** [reply]
I'm not aware that anyone is working on this.

**Reply by AnirvanB** [reply]
Got it! So, I'm new to LabWC. Is there a place where we can check what features or enhancements will be included in the upcoming launch milestones or where we can vote for their inclusion?

**Reply by 01micko** [reply]
A good place to look would be https://github.com/labwc/labwc-scope

Can't say there is a place to vote there but anyone can raise an issue.

**Reply by AnirvanB** [reply]
Thanks @01micko! Will check it out.

**Comment by 01micko** [comment]
If handle/grip are added it would be a good idea to add support for rounded bottom corners at the same time.
